public-api/src/main/java/org/pmiops/workbench/cdr/model/QuestionConcept.java

```diff
@@ -131,6 +131,7 @@ public static void setRaceStratumNameMap() {
 
     public static void setRaceEthnicityStratunNameMap() {
         raceEthnicityStratumNameMap.put("903070", "Other");
+        raceEthnicityStratumNameMap.put("903096", "Skip");
```

Comment:
Prod data has skip answer and without this it shows weird name for 903096 in graphs.